pkg/apis/networking/v1alpha1/service_types.go

```diff
@@ -53,16 +53,33 @@ type MultiClusterServiceSpec struct {
 	// +optional
 	Ports []ExposurePort `json:"ports,omitempty"`
 
-	// Range specifies the ranges where the referencing service should
-	// be exposed.
-	// Only valid and optional in case of Types contains CrossCluster.
-	// If not set and Types contains CrossCluster, all clusters will
-	// be selected, that means the referencing service will be exposed
-	// across all registered clusters.
+	// DiscoveryStrategy specifies how to sync the EndpointSlice from one cluster to another cluster.
+	// Default to be RemoteAndLocal, which means syncing the EndpointSlice from one cluster
+	// to another cluster from the beginning, the requests to this service will routed to all of them.
 	// +optional
-	Range ExposureRange `json:"range,omitempty"`
```

Comment:
Please don't just remove the field even though it has never been used.
See [Association in PropagationPolicy](https://github.com/karmada-io/karmada/blob/186e137eecde2ca63140265c79fa6a5c7946ab5d/pkg/apis/policy/v1alpha1/propagation_types.go#L68-L73) for example.